docs/quickstart.ipynb

```diff
@@ -116,7 +116,7 @@
     "sim = run_tardis(\"tardis_example.yml\", \n",
     "                 virtual_packet_logging=True,\n",
     "                 show_convergence_plots=True,\n",
-    "                 export_convergence_plots=True,\n",
+    "                 export_convergence_plots=False, #TODO: to avoid double plots\n",
```

Comment:
Maybe we need to replace this parameter with a new parameter (maybe called `export_static_output`) which can show convergence plots and logger-widget (or atleast a fixed height html text area) for static HTML documentation website.

And setting this param to True should be controllable by identifying if the notebook execution is being done by nbsphinx/nbconvert - there should be some setting to detect that but needs research.